.github/workflows/nvhpc-cmake.yml

```diff
@@ -72,13 +77,13 @@ jobs:
       - name: CMake Run Tests
         shell: bash
         run: |
-          ctest . -E "MPI_TEST|H5TEST-dt_arith|H5TEST-dtransform" --parallel 2 -C ${{ inputs.build_mode }} -V
+          ctest . -E "MPI_TEST|H5TEST-dt_arith" --parallel 2 -C ${{ inputs.build_mode }} -V
```

Comment:
@brtnfld Are we not testing parallel with NVHPC?

Reply:
Parallel is tested later down